spyder/plugins/ipythonconsole.py

```diff
@@ -861,12 +861,19 @@ def create_new_client(self, give_focus=True):
         """Create a new client"""
         self.master_clients += 1
         name = "%d/A" % self.master_clients
+        cf = self._new_connection_file()
+        if cf is None:
+            QMessageBox.warning(self, _('Warning'),

```

Comment:
@ccordoba12 I already fixed the <br>, however I can't use the ```client.show_kernel_error``` because in order to create the client the conecction file has to be not None. Is there another option to not use ```QMessageBox.warning```?